module/VuFind/src/VuFind/Db/Table/User.php

```diff
@@ -104,6 +104,19 @@ public function createRowForUsername($username)
         return $row;
     }
 
+    /**
+     * Retrieve a user object from the database based on the given field name.
+     *
+     * @param string $fieldName  Field name
+     * @param string $fieldValue Field value
+     *
+     * @return UserRow
```

Comment:
Should this return `?UserRow` in case no match is found?

Reply:
Yes, i also changed it for the other getter methods.